### PR TITLE
Support the new alpaca data source

### PR DESCRIPTION
**Pull request by shlomikushchi**
In this PR we integrate the new Alpaca data API
we support these:
* getLastQuote
* getLastTrade
* get minute bars
* websocket for live udpates

### CODE REVIEW

File: lib/resources/entity.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    "as": "asksize",
+    "bx": "bidexchange",
+    "bp": "bidprice",
+    "bs": "bidsize",
+    "c": "condition",
+    "t": "timestamp"
+};
+
+function AlpacaQuote(data) {
+    let obj = {};
```
Comment (umitanuki) [inline lib/resources/entity.js:64]: I believe it's usually const

File: lib/resources/entity.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+        if (alpaca_quote_mapping.hasOwnProperty(key)) {
+            obj[alpaca_quote_mapping[key]] = value;
+        } else {
+            obj[key] = value;
+        }
+    }
+    return obj;
+}
+
+function AlpacaTrade(data) {
```
Comment (umitanuki) [inline lib/resources/entity.js:75]: Looks like you don't need to repeat the same pattern but just have curried functions. Just an idea.

Comment (shlomikushchi) [reply lib/resources/entity.js:67]: done

File: lib/resources/entity.js (hunk 1 above)
Comment (shlomikushchi) [reply lib/resources/entity.js]: ok

**Pull request merged**